keras/applications/mobilenetv2.py

```diff
@@ -0,0 +1,535 @@
+"""MobileNet v2 models for Keras.
+
+MobileNetv2 is a general architecture and can be used for multiple use cases.
```

Comment:
Would it be better to make these words (`MobileNet v2`, `MobileNetv2`, `MobileNetV2`) consistent?